### PR TITLE
Feature restaurant-page

### PR DESCRIPTION
**Pull request by IjayAbby**
This is a  restaurant page built with Vanilla JS with implementation of webpack configuration and styling of the same.

### CODE REVIEW

File: src/contact.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+function createContact() {
```
Comment (addod19) [inline src/contact.js]: Kindly use the ES6 arrow function instead :muscle:

File: src/contact.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  const address = document.createElement('p');
+  address.textContent = 'Addres: Nairobi-Kisumu Highway, In the Mega City Mall compound., Kisumu, KE 40100';
+
+  contact.appendChild(phoneNumber);
+  contact.appendChild(address);
+
+  return contact;
+}
+
+function loadContact() {
```
Comment (addod19) [inline src/contact.js]: Kindly use the ES6 arrow function instead

File: src/home.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+function createParagraph(text) {
+  const paragraph = document.createElement('p');
+  paragraph.textContent = text;
+  return paragraph;
+}
+
+function createHome() {
```
Comment (addod19) [inline src/home.js]: Kindly use the ES6 arrow function instead

File: src/home.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  home.classList.add('home');
+
+  home.appendChild(createParagraph('Best african food in Kenya'));
+  home.appendChild(createParagraph('Made with love'));
+  home.appendChild(createParagraph('Order online or visit us!'));
+
+  return home;
+}
+
+function loadHome() {
```
Comment (addod19) [inline src/home.js]: Kindly use the ES6 arrow function instead

File: src/menu.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
+function createMenu() {
```
Comment (addod19) [inline src/menu.js]: Kindy use the ES6 arrow function instead

File: src/menu.js [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    createMenuItem(
+      'Ugali',
+      "This is Kenya's staple food. Mostly served with stews like beef, chicken or fish.",
+    ),
+  );
+
+  return menu;
+}
+
+function createMenuItem(name, description) {
```
Comment (addod19) [inline src/menu.js]: Kindly use the ES6 arrow function instead

File: src/menu.js [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+  foodImage.alt = `${name}`;
+
+  menuItem.appendChild(foodImage);
+  menuItem.appendChild(foodName);
+  menuItem.appendChild(foodDescription);
+
+  return menuItem;
+}
+
+function loadMenu() {
```
Comment (addod19) [inline src/menu.js]: Kindly use the ES6 arrow function instead

File: src/website.js [hunk 8]
```diff
@@ -0,0 +1,108 @@
+import loadHome from './home';
+import loadMenu from './menu';
+import loadContact from './contact';
+
+function createHeader() {
```
Comment (addod19) [inline src/website.js]: Kindly use the ES6 arrow function instead

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @IjayAbby, congratulations on this project :+1: The entry point has been set up nicely :smile: The contents of each tab are stored in a module :100: Each module exports a function that creates a `div` elements, adds the appropriate content and styling :100: 
Let us address the comments below so we can get our project approved:

- Kindly add `webpack-dev-server` to the devDependencies, because we cannot use the `serve` command without installing the webpack-dev-server as in the screenshot below:
code2
- Kindly use the recommended `.eslintrc.json` file, do not make any changes. You can find the recommended file [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)
- Kindly use the ES6 arrow function instead of the `function` keyword. Kindly check this for all functions
- Kindly fix the broken images on the Menu page

Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by afanimartin — APPROVED** [review]
## Approved 👏🏿 
Nice work making the requested changes. Merge and proceed to the next project. Good luck!

**Pull request merged**